config/manager/manager.yaml

```diff
@@ -51,39 +51,5 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        - name: ADDON_NAME
-        - name: SOP_ENDPOINT
-        - name: ALERT_SMTP_FROM_ADDR
-        - name: DEPLOYMENT_TYPE
-        - name: RHOBS_ENDPOINT
-        - name: RH_SSO_TOKEN_ENDPOINT
-      - name: readiness-server
-        securityContext:
-          runAsNonRoot: true
-          allowPrivilegeEscalation: false
-          capabilities:
-            drop:
-              - ALL
-        command:
-        - /readinessServer
-        image: controller:latest
-        readinessProbe:
-          httpGet:
-            path: /readyz
-            port: 8081
-          initialDelaySeconds: 5
-          periodSeconds: 10
-        resources:
-          limits:
-            cpu: 100m
-            memory: 30Mi
-          requests:
-            cpu: 100m
-            memory: 30Mi
-        env:
-        - name: NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
       terminationGracePeriodSeconds: 10
-      serviceAccountName: deployer
+      serviceAccountName: agent
```

Comment:
We don't need to add managed-fusion manually, it will get prefixed based on whatever namePrefix you have mentioned in kustomization.yaml.
In this scenario, we have mentioned `managed-fusion-` so the final service account name in bundle will be `managed-fusion-agent`.
https://github.com/red-hat-storage/managed-fusion-agent/pull/3/files#diff-c006b1ef995eeeb57408cce42bd84db8dfc0cbe18df03222857665cebb1ac6fd

Reply:
@bindrad In that case, we need to revert the changes back to agent

Reply:
Updated in latest commit